api/src/main/java/org/pmiops/workbench/db/dao/UserEgressBypassWindowDao.java

```diff
@@ -6,5 +6,5 @@
 
 public interface UserEgressBypassWindowDao extends CrudRepository<DbUserEgressBypassWindow, Long> {
   // The extra "By" infix is necessary in combination with OrderBy...
-  Set<DbUserEgressBypassWindow> getByUserIdOrderByStartTimeDesc(long userId);
+  Set<DbUserEgressBypassWindow> getByUserIdOrderByStartTimeDesc(Long userId);
```

Comment:
because this is called using a `Long` we need to specify that non-primitive type here too, or we can get an unboxing NPE.  Added test for this case, which faisl on main.